etc/lamassu/application.properties

```diff
@@ -49,6 +49,9 @@ org.entur.lamassu.targetGbfsVersion=2.3
 ## Base URL for API (needed for URL transformations)
 org.entur.lamassu.baseUrl=${LAMASSU_BASE_URL}
 
+# How often should feed be updated (in case it's ttl is exceeded), in ms, default 6h
```

Comment:
What does "in case it's ttl is exceeded" mean?

Reply:
"ttl" is the gbfs feed's time to live property. In case current time is past lastUpdate+ttl, lamassu will refetch the feed.

Reply:
How does this relate to a static interval? Does this mean that the feed will be refetched *earlierst* when `currentTime > lastUpdate + ttl`, or *latest`?

Reply:
Earliest. Lamassu only checks every `org.entur.lamassu.feedupdateinterval` milliseconds